java.jdbc-ballerina/Module.md

```diff
@@ -0,0 +1,89 @@
+## Overview
+
+This module provides the functionality required to access and manipulate data stored in any type of relational database
+that is accessible via Java Database Connectivity (JDBC).
+
+**Prerequisite:** Add the JDBC driver corresponding to the database you are trying to interact with
+as a native library dependency in your Ballerina project. Then, once you build the project by executing the `ballerina build`
+command, you should be able to run the resultant by executing the `ballerina run` command.
+
+E.g., The `Ballerina.toml` content for h2 database.
+Change the path to the JDBC driver appropriately.
+
+```toml
+[package]
+org = "sample"
+name = "jdbc"
+version= "0.1.0"
+
+[[platform.java11.dependency]]
+artifactId = "h2"
+version = "1.4.200"
+path = "/path/to/com.h2database.h2-1.4.200.jar"
```

Comment:
I think we don't need to add h2 jar path as the H2 database driver JAR is included by default in the Ballerina distribution.
Do we need to add this?

Reply:
What if the developer needs to use a different version of h2 database. this `ballerina.toml` explanation would be helpful for  the developer